.github/actions/build-variant/action.yml

```diff
@@ -0,0 +1,97 @@
+name: Setup Build Variant Composite Action
+description: Variant build actions for Meshtastic Platform IO steps
+
+inputs:
+  board:
+    description: The board to build for
+    required: true
+  github_token:
+    description: GitHub token
+    required: true
+  build-script-path:
+    description: Path to the build script
+    required: true
+  remove-debug-flags:
+    description: A newline separated list of files to remove debug flags from
+    required: false
+    default: ""
+  ota-firmware-source:
+    description: The OTA firmware file to pull
+    required: false
+    default: ""
+  ota-firmware-target:
+    description: The target path to store the OTA firmware file
+    required: false
+    default: ""
+  artifact-paths:
+    description: A newline separated list of paths to store as artifacts
+    required: false
+    default: ""
+  include-web-ui:
+    description: Include the web UI in the build
+    required: false
+    default: "false"
+
+runs:
+  using: composite
+  steps:
+    - uses: actions/checkout@v4
+
+    - name: Build base
+      id: base
+      uses: ./.github/actions/setup-base
+
+    - name: Pull web ui
```

Comment:
In one of the tests just now "Pull web ui" failed, but there was no log explaining as to why. Just wondering, is there any easy config you can add here that would give a bit more debug info? If not easy, please ignore this comment :)

Reply:
Looks like a rerun resolved it. That one was weird, there were no logs for the one run, but there were for other runs